vita3k/modules/module_parent.cpp

```diff
@@ -129,6 +150,7 @@ void call_import(EmuEnvState &emuenv, CPUState &cpu, uint32_t nid, SceUID thread
 
             if (!emuenv.missing_nids.contains(nid) || LOG_UNK_NIDS_ALWAYS) {
                 LOG_ERROR("Import function for NID {} not found (thread name: {}, thread ID: {})", log_hex(nid), thread->name, thread_id);
+                LOG_DEBUG("{}\n{}", save_context(*thread->cpu).description(), thread->log_stack_traceback());
```

Comment:
This is a traceback log I didn't remove.

Reply:
Oh sorry lol oki

Reply:
Done